model/src/test/kotlin/licenses/LicenseClassificationsTest.kt

```diff
@@ -112,6 +113,14 @@ class LicenseClassificationsTest : WordSpec({
             }
         }
 
+        "contain empty categories" {
```

Comment:
Better "maintain" instead of "contain", as the latter sounds like "any arbitrary empty categories".